hq/test/logic/SnykDisplayTest.scala

```diff
@@ -8,7 +8,7 @@ import scala.concurrent.ExecutionContext.Implicits.global
 
 class SnykDisplayTest extends FreeSpec with Matchers with AttemptValues {
 
-  private def readFile(filename:String) = Source.fromResource(s"logic/SnykDisplayTest/$filename.json").getLines.mkString
+  private def readFile(filename:String) = Source.fromResource(s"logic/SnykDisplayTestResources/$filename.json").getLines.mkString
```

Comment:
This fixes a compile error. The compiler was unhappy, because both the test and the file with the resources had the same name.